syntax = "proto3";
package custom_sender;




import "model/msgsender/send_message_request_data.proto";
import "google/protobuf/struct.proto";

/*
SendMessage返回
*/
message SendMessageResponse {
    /*
    请求结果数据 e.g.: {"email": ["test@qq.com"], "phone": ["test@qq.com"]}
    */
    google.protobuf.Struct data = 1 ;
    /*
    请求错误信息
    */
    string msg = 2 ;
    /*
    请求返回码
    */
    int32 code = 3 ;
}

/*
SendMessageApi返回
*/
message SendMessageResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    custom_sender.SendMessageResponse data = 4 ;
}

